src/main/scala/org/bitcoins/core/script/locktime/LockTimeInterpreter.scala

```diff
@@ -158,18 +159,18 @@ trait LockTimeInterpreter extends BitcoinSLogger {
     // unless the type of nSequenceMasked being tested is the same as
     // the nSequenceMasked in the transaction.
     if (!(
-      (txToSequenceMasked < ScriptNumber(TransactionConstants.sequenceLockTimeTypeFlag) &&
-        nSequenceMasked < ScriptNumber(TransactionConstants.sequenceLockTimeTypeFlag)) ||
-        (txToSequenceMasked >= ScriptNumber(TransactionConstants.sequenceLockTimeTypeFlag) &&
-          nSequenceMasked >= ScriptNumber(TransactionConstants.sequenceLockTimeTypeFlag))
+      (txToSequenceMasked.underlying < TransactionConstants.sequenceLockTimeTypeFlag &&
```

Comment:
Instead of calling `.underlying` here, wrap `TransactionConstants.sequenceLockTimeTypeFlag` in a Int64